dask_xgboost/core.py

```diff
@@ -68,7 +67,7 @@ def concat(L):
     elif isinstance(L[0], (gd.DataFrame, gd.Series)):
         return gd.concat(L)
     elif isinstance(L[0], xgb.DMatrix):
-        return L
+        return
```

Comment:
Was the `return L` meant to be dropped here?

Reply:
Yes. Technically, this logical block doesn't do anything. The `data` is tested under `isinstance(data, xgb.DMatrix)` in the `train_part` function, and when it passes, no `concat` is done. This makes it explicit that `concat(xgb.DMatrix)` should never be called.